torch/_tensor_str.py

```diff
@@ -266,6 +266,20 @@ def _str(self):
         if values.numel() == 0:
             values_str += ', size=' + str(tuple(values.shape))
         tensor_str = indices_prefix + indices_str + '),\n' + ' ' * indent + values_prefix + values_str + ')'
+    elif self.is_quantized:
+        suffixes.append('size=' + str(tuple(self.shape)))
+        suffixes.append('dtype=' + str(self.dtype))
+        suffixes.append('quantization_scheme=' + 'per_tensor_affine')
+        suffixes.append('scale=' + str(self.q_scale()))
```

Comment:
wait, why are these tensors at the python level?

Reply:
I don't know..It is Scalar in API, but do we have Variable -> Scalar? I suppose self.q_scale() is a variable?

Reply:
I think you can just do `self.q_scale().item()`.